src/test/java/sorinfoca/Tests/Tema6/HerokuForms.java

```diff
@@ -0,0 +1,75 @@
+package sorinfoca.Tests.Tema6;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import sorinfoca.driver.BrowserManager;
+import java.io.File;
+import java.io.IOException;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws InterruptedException {
+        navigateToHerokuHomePage();
+        try {
+            fillOutForm();
+            scrollToSubmitButton();
+            submitForm();
+        } catch (NoSuchElementException e) {
+            File screenshot = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
```

Comment:
All the lines written in this catch clause are already written in FileUtils.takeScreenshot() method. Use that method instead of all these lines as used in HerokuAlerts class. Thanks!